hazelcast-code-generator/src/main/resources/codec-template-md.ftl

```diff
@@ -333,3 +335,16 @@ Header only event message, no message body exist.
     </#switch>
 </#function>
 
+
+<#function resolvePartitionIdentifier partitionIdentifier>
+    <#switch partitionIdentifier?trim>
+        <#case "random">
+            <#return "a random partition id from 0 to PARTITION_COUNT(inclusive)">
```

Comment:
Partition count is configurable and its default is written somewhere in document.
It was written 271. I recently changed it as stating it is the default value and it is configurable.

Here is the related part after my change.
```
By default there are 271 partitions. Note that this value can be configured in server side. Partition id s starts from zero. For 271 partitions, ids are 0 to 270
```

I just realize that inclusive is wrong here. It should be exclusive. I will also change that with direct commit.